tests/test_slogpy_console.py

```diff
@@ -0,0 +1,24 @@
+"""Import slogpy from slog"""
+from slogpy.slog import Slog as slog  # noqa: N813
+
+"""This is a setup function"""
+def setup_function():
+    """set "file_logging = False", so it will not create a log file for output."""
+    slog.initialize(file_logging=False)
+
+"""First unit test"""
+def test_info_console_output(capsys, msg="This is a console test"):
+    """Passing a message to the info method and then capture the message from the console"""
+    slog.info(msg)
+    expected_content = f"{msg}\n"
+    out, _ = capsys.readouterr()
+    assert expected_content == out
+
+"""Second unit test (debug method)"""
+def test_debug_console_output(capsys, msg=""):
+    """Passing a message to the Debug method and then capture the message from the console"""
+    slog.debug(msg)
+    expected_content = f"{msg}"
+    # not sure this line of code is necessary or not. since this won't be any content in the console.
+    out, _ = capsys.readouterr()
+    assert expected_content == ""
```

Comment:
question: did this test actually pass?
line 21 sets expected_content to a non-empty string.  This line is checking that the expected_content is empty.  It should fail.

Reply:
Yes, the test did pass.

Reply:
![pytest_console](https://github.com/slogpy/slogpy/assets/116843236/1ea00607-ba2f-4f6c-b25a-b04a88544829)